app/models/test_track/lazy_visitor_by_identity.rb

```diff
@@ -14,15 +14,19 @@ def id_loaded?
   private
 
   def method_missing(method, *args, &block)
-    __visitor__.send(method, *args, &block)
+    if __visitor__.respond_to?(method)
+      __visitor__.send(method, *args, &block)
+    else
+      super
```

Comment:
Rubocop wants you to fall back on `super` from method_missing. I get why it's valuable in most cases. In this case, it means that we get a slightly different stacktrace when a method truly isn't defined on the visitor class. 🤔